app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -53,6 +53,9 @@ interface ICloneRepositoryState {
   /** The local path to clone to. */
   readonly path: string
 
+  /** A copy of the initial local path to clone to. */
```

Comment:
Could we elaborate in this documentation and explain why we're keeping a copy of the path? To a new contributor or someone who isn't familiar with the problem this PR is trying to solve the current documentation is only providing information on what it is, not why it's there or what it's used for.